packages/aws-cdk/lib/cli/cdk-toolkit.ts

```diff
@@ -206,6 +207,15 @@ export class CdkToolkit {
     await this.props.configuration.saveContext();
   }
 
+  public async cliTelemetryStatus() {
+    const currentStatus = canCollectTelemetry(this.props.configuration.context);
+    if (currentStatus) {
+      await this.ioHost.asIoHelper().defaults.info('CLI Telemetry is enabled. Run \'cdk cli-telemetry --disable\' to disable for this CDK App.');
```

Comment:
Maybe link to the README so that the user is also exposed to the other mechanisms?